proto/repositories.bzl

```diff
@@ -17,24 +17,17 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 load("//proto/3rdparty/crates:defs.bzl", "crate_repositories")
+load("//proto/prost:repositories.bzl", "rust_prost_dependencies", "rust_prost_register_toolchains")
 
-# buildifier: disable=unnamed-macro
-def rust_proto_repositories(register_default_toolchain = True):
```

Comment:
You should probably keep `rust_proto_repositories` to not break existing users, but I do like the new consistency in the two new macros!

Reply:
Oops! Indeed it is 😅